lib/elixir/lib/gen_server.ex

```diff
@@ -159,6 +159,39 @@ defmodule GenServer do
   the GenServer callbacks as doing so will cause the GenServer to misbehave.
   If you want to receive custom messages, always receive them in `handle_info/2`.
 
+  Custom messages is all other messages a server may receive that are not sent via
+  [`GenServer.call/3`](http://elixir-lang.org/docs/stable/elixir/GenServer.html#call/3),
+  or [`GenServer.cast/2`](http://elixir-lang.org/docs/stable/elixir/GenServer.html#cast/2).
```

Comment:
You don't need to use links here, ExDoc will figure the link out if you just write

```
... `call/3` or `cast/2`
```

As you can see you don't even need to use the fully qualified name (`GenServer.*`) since you're in the `GenServer` module :)